tests/container_disk_test.go

```diff
@@ -176,12 +176,16 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting from custom image location", func() {
 		Context("with disk at /custom-disk/downloaded", func() {
 			It("[test_id:1466]should boot normally", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirrosCustomLocation), "#!/bin/bash\necho 'hello'\n")
-				for ind, volume := range vmi.Spec.Volumes {
-					if volume.ContainerDisk != nil {
-						vmi.Spec.Volumes[ind].ContainerDisk.Path = "/custom-disk/downloaded"
-					}
-				}
+				vmi := libvmi.New(
+					libvmi.WithResourceMemory("256Mi"),
```

Comment:
I'm not very happy with this unexplained hard-coded value. Can you at least set it as a const? Or even move th evil `cirrosMemory` function to this file.

Reply:
A const is enough IMO. We have zero interest in optimizing for 128M of memory for a a few incidents. Going with the worse case should be good enough.